pkg/env/env.go

```diff
@@ -7,16 +7,25 @@ import (
 	"bufio"
 	"fmt"
 	"os"
+	"path"
 	"strings"
 
 	"github.com/pkg/errors"
 )
 
 // DefaultEnvVariables set $PATH, $TERM and $container.
 var DefaultEnvVariables = map[string]string{
-	"PATH":      "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
-	"TERM":      "xterm",
-	"container": "podman",
+	"PATH": "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
+	"TERM": "xterm",
+}
+
+func init() {
+	// Set "container" to the current binary.
+	bin, err := os.Executable()
```

Comment:
do we need to guard against symlinks here?

Reply:
Good question. We don't really need to protect us from them as we only use it as a string but if we had a `docker -> podman` symlink, we would add `docker` as the value. I see a beauty in both.

What's your preference?

Reply:
I am not sure that is true.  It would depend on how you did the symbolic link.